### ISSUE TITLE
Update Visual Studio Version

### ISSUE THREAD

**Issue by lock9** [neo-project/neo-devpack-dotnet#39]
Hello,
The current DevPack is not compatible with the latest version of the compiler.

I spoke with @igormcoelho  and it seems that the version I'm using is outdated, but it is the latest available in VisualStudio.
Here is the error:
```C#
Neo.Compiler.MSIL console app v2.3.0.9
Convert Error:System.Exception: error:System.Void SimpliPay.BankDao::PersistRegularAccount(System.Byte[],System.Byte[])::IL_0007 Call System.Byte[] Neo.SmartContract.Framework.Helper::Concat(System.Byte[],System.Byte[]) ---> System.InvalidCastException: Unable to cast object of type 'System.Byte' to type 'Mono.Cecil.CustomAttributeArgument[]'.
   at Neo.Compiler.MSIL.ModuleConverter.IsOpCall(MethodDefinition defs, OpCode[]& opcodes)
   at Neo.Compiler.MSIL.ModuleConverter._ConvertCall(OpCode src, NeoMethod to)
   at Neo.Compiler.MSIL.ModuleConverter.ConvertCode(ILMethod method, OpCode src, NeoMethod to)
   at Neo.Compiler.MSIL.ModuleConverter.ConvertMethod(ILMethod from, NeoMethod to)
   --- End of inner exception stack trace ---
   at Neo.Compiler.MSIL.ModuleConverter.ConvertMethod(ILMethod from, NeoMethod to)
   at Neo.Compiler.MSIL.ModuleConverter.Convert(ILModule _in, ConvOption option)
   at Neo.Compiler.Program.Main(String[] args)
```

Thanks

**Comment by lock9** [comment]
I would to ask you guys to please create a development branch too, if possible. Thanks

**Comment by igormcoelho** [comment]
in fact these last changes are still under development, you probably have the latest version (which is released). Dev branch may be a good idea perhaps, to avoid confusions.

**Comment by lock9** [comment]
The compiler was fixed but I can't use it because this. @igormcoelho  do you know who is responsible for publishing this into VS?

**Comment by erikzhang** [comment]
I just released the latest version (v2.9.3) of the plugin and devpack.
